manifests/a/AGSProjectTeam/AdventureGameStudio/3.5.1.19/AGSProjectTeam.AdventureGameStudio.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AGSProjectTeam.AdventureGameStudio
+PackageVersion: 3.5.1.19
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/adventuregamestudio/ags/releases/download/v.3.5.1.19/AGS-3.5.1.19-P12.exe
+  InstallerSha256: 9C2C1BE2B577C69EF40F9FD43048BA4FD17D0C4383F9AFDE00951BC3DB182BAE
```

```suggestion
  InstallerSha256: 9C2C1BE2B577C69EF40F9FD43048BA4FD17D0C4383F9AFDE00951BC3DB182BAE
  AppsAndFeaturesEntries: 
  - DisplayName: Adventure Game Studio 3.5.1 P12
```
ARP Entry for better package correlation